libsupport/src/ArrowInterchange.cpp

```diff
@@ -57,3 +57,17 @@ katana::Shuffle(const std::shared_ptr<arrow::ChunkedArray>& original) {
   KATANA_LOG_ASSERT(res.ok());
   return IndexedTake(original, indices);
 }
+
+std::shared_ptr<arrow::ChunkedArray>
+katana::EmptyChunkedArray(
+    const std::shared_ptr<arrow::DataType>& type, int64_t length) {
+  // Creating an int64 array, but this array shouldn't be used directly
```

Comment:
Stale comment? I think you've parameterized the type if I understand this code correctly.

Reply:
Busted!